packages/ui/src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { MouseEventHandler } from 'react';
+import { forwardRef, HTMLChakraProps, Image as ChakraImage } from '@chakra-ui/react';
+
+import blackHorizontalLogo from '../../../images/LogoBlackHorizontal.svg';
+import blackSquareLogo from '../../../images/LogoBlackSquare.svg';
+import whiteHorizontalLogo from '../../../images/LogoWhiteHorizontal.svg';
+import whiteSquareLogo from '../../../images/LogoWhiteSquare.svg';
+
+type LayoutVariant = 'horizontal' | 'square';
+
+type LogoColor = 'white' | 'black';
+
+export interface LogoProps extends HTMLChakraProps<'img'> {
+  layout?: LayoutVariant;
+  color?: LogoColor;
+  onClick?: MouseEventHandler;
+}
+
+const getSrcFromProps = (color: LogoColor, layout: LayoutVariant): string => {
```

Comment:
I would consider using an object map instead of a function. It could look like below:
```
const logosMap = {
    white: { horizontal: whiteHorizontalLogo, square:  whiteSquareLogo },
    black: { horizontal: blackHorizontalLogo, square:  blackSquareLogo },
}
```
In such a case, you will use `logosMap[color][layout]` as the value. It would remove the need to return`'./LogoWhiteHorizontal'` as a default value.

Reply:
done